css/style.css

```diff
@@ -0,0 +1,41 @@
+body {
+}
```

Comment:
We should not use empty rulesets.

Reply:
I just forgot to add what I wanted to add to be honest